### PR TITLE
interp: prevent an off-by-one during interp debug

### PR DESCRIPTION
**Pull request by dgryski**

### CODE REVIEW

File: interp/interpreter.go [hunk 1]
```diff
@@ -493,8 +493,8 @@ func (r *runner) run(fn *function, params []value, parentMem *memoryView, indent
 				// possibly trying to recover from it if it failed to execute.
 				if r.debug {
 					argStrings := make([]string, len(operands))
```
Comment (QuLogic) [inline interp/interpreter.go]: This should be `len(operands)-1`, I think, or you get a trailing comma, unless that's desired for some reason.

Comment (dgryski) [reply interp/interpreter.go]: Oh, you're right. I fixed the out-of-bounds crash but didn't then check the output was correct.

Comment (dgryski) [reply interp/interpreter.go]: Fixed off-by-one in the off-by-one fix.

**Comment by deadprogram** [comment]
Looking good, thanks @dgryski for the fix and @QuLogic for the review. Now merging.

**Pull request merged**